packages/main/test/pages/TextArea.html

```diff
@@ -29,6 +29,16 @@
 
 	<body class="textarea1auto">
 
+	<section class="group">
+		<ui5-title>Single Row TextArea with Exceeded Characters</ui5-title>
+		<br>
+		<ui5-textarea id="textarea-single-row"
+					  show-exceeded-text="true"
+					  growing-max-lines="0"
```

Comment:
Growing property is false by default, so without adding it, growing-max-lines does not influence the component. Also, 0 is the default value, so you do not have to specify it.